tools/gyp/pylib/gyp/input.py

```diff
@@ -2040,7 +2047,7 @@ def MakePathRelative(to_file, fro_file, item):
         gyp.common.RelativePath(os.path.dirname(fro_file),
                                 os.path.dirname(to_file)),
                                 item)).replace('\\', '/')
-    if item[-1] == '/':
+    if item[-1:] == '/':
```

Comment:
Is this required? A longer string cannot be equal to a single character anyway? Are we unnecessarily changing a cmp op into a strcmp op?

Reply:
Introduced in https://github.com/nodejs/node-gyp/commit/47496bf6dcf09d3fc95fdd069864eda36e094f56

Reply:
Now I understand.  This should stay as is.

Reply:
__if item.endswith('/'):__